java/buildconf/build-props.xml

```diff
@@ -38,6 +38,7 @@
 
   <!-- =================== jar dependencies ======================= -->
   <property name="commons-lang" value="commons-lang3"/>
+  <property name="commons-text" value="commons-text"/>
```

Comment:
the variable is not strictly required. I think commons-lang3 has it just because in the past we had different names on different OS versions apache vs jakarta etc. 
Would be ok to just write commons-text directly into the other properties directly